rpm-check.sh

```diff
@@ -387,6 +387,13 @@ check_single_file()
           sed -i -e 's%<td>[A-Z][a-z][a-z] [A-Z][a-z][a-z] [0-9]\+ [0-9]\+:[0-9]\+:[0-9]\+ +0000 201[0-9]</td>%<td>Mon Sep 20 19:02:43 +0000 2010</td>%g' $f
        done
        ;;
+    *adaptor.h)
+       # qdbusxml2cpp generates timestamps in the include guards
+       # #ifndef SOMETHINGADAPTOR_H_1442211518
+       # #define SOMETHINGADAPTOR_H_1442211518
```

Comment:
qdbusxml2cpp also generates *interface.h files which have include guards SOMETHING_INTERFACE_H_1442211518. Is it appropriate to handle those files here too?

This has been fixed upstream in Qt [1]. Cherry-picking [1] into the Mer Qt build would make this change mostly unnecessary.

[1] http://code.qt.io/cgit/qt/qtbase.git/commit/?id=c030540f37084effca4aa7e8d94ceeda95d60487